src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/ProjectConfigurationStateSynchronizerTest.cs

```diff
@@ -63,14 +64,14 @@ public async Task ProjectConfigurationFileChanged_Removed_NonNormalizedPaths()
             .Setup(x => x.AddProject(
                 projectInfo.FilePath,
                 intermediateOutputPath,
-                projectInfo.Configuration,
+                It.IsAny<RazorConfiguration>(),
```

Comment:
For the curious or inquisitive person who finds this, I had to change this to `IsAny` because we're mocking our own code and the path to add/update a project enforces that the configuration is up to date with the feature flags. That causes reference equality to fail and thus moq to fail.

Reply:
😭